doc/api/fs.markdown

```diff
@@ -801,6 +801,10 @@ on Unix systems, it never was.
 
 Returns a new ReadStream object (See `Readable Stream`).
 
+Be aware that, unlike the default value set for `highWaterMark` on a
+readable stream(16kb), the stream returned by this method has a
```

Comment:
Space before paren?  Also, isn't it more common (and correct) to write it as `16 kB`?